src/xdist/scheduler/loadscope.py

```diff
@@ -1,18 +1,14 @@
-from collections import OrderedDict
-
-from _pytest.runner import CollectReport
+from .load import LoadScheduling
 from py.log import Producer
-from xdist.report import report_collection_diff
-from xdist.workermanage import parse_spec_config
 
 
-class LoadScopeScheduling(object):
+class LoadScopeScheduling(LoadScheduling):
     """Implement load scheduling across nodes, but grouping test by scope.
```

Comment:
So `loadfile` and `loadscope` are distribute the tests identically now? Or am I missing something?